diff --git a/components/permissions/android/permission_dialog_delegate.cc b/components/permissions/android/permission_dialog_delegate.cc
index 4de29c5376b001a7b3b025263fdf224bec57aaf7..ea74aece42f9a7026c77faf10d8b2d898ba1d45c 100644
--- a/components/permissions/android/permission_dialog_delegate.cc
+++ b/components/permissions/android/permission_dialog_delegate.cc
@@ -67,6 +67,7 @@ void PermissionDialogDelegate::CreateJavaDelegate(
 
 void PermissionDialogDelegate::Accept(JNIEnv* env,
                                       const JavaParamRef<jobject>& obj) {
+  BRAVE_PERMISSION_DIALOG_DELEGATE_ACCEPT
   permission_prompt_->Accept();
 }
 
